core/src/win32/filed/vss_generic.cc

```diff
@@ -1128,7 +1128,7 @@ bool VSSClientGeneric::CheckWriterStatus()
     return false;
   }
 
-  int nState;
+  int nState = 0;
```

Comment:
this doesn't make sense here. Move after 1149 or even better wrap lines 1150 to 1180 into an IIFE.

Reply:
Will remove this change.